bundle-workflow/src/bwc_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import sys
+
+from manifests.bundle_manifest import BundleManifest
+from system.temporary_directory import TemporaryDirectory
+from test_workflow.bwc_test_suite import BwcTestSuite
+from test_workflow.test_args import TestArgs
+
+
+def main():
```

Comment:
I think this should be a BwcTestSuite class with an `.execute` method that takes `TestArgs`. The only way to call this would be `test.sh`.